docs/advanced_topics/security_considerations.md

```diff
@@ -0,0 +1,111 @@
+# Security considerations
```

Comment:
We should give this page a reference and then add a cross-link to it in `docs/topics/writing_templates.md`. Where we talk about "For sites enforcing a Content Security Policy...".